### DISCUSSION TITLE
Set 169MHz

### DISCUSSION THREAD

**Discussion by pe1evx** [jgromes/RadioLib#833] [Q&A] [answered]
Hello all,

I am using a RFM98P and an Arduino Nano. I can receive POCSAG calls on 434MHz from my own generator. But this RFM98P is designed (frontend) for 169MHz. Now I want to change it, but the library seems to refuse this frequency. The chipset should be able to do so. Any suggestions where the band limits are implemented in the library?

Many thanks in advance.

Alex PE1EVX

**Comment by jgromes** [comment, marked as answer]
The RFM9x datasheet shows this frequency range, and the range checks in RadioLib are set according to that. Where did you get the information that it should support 169 MHz?

![Screenshot_212](https://github.com/jgromes/RadioLib/assets/3407192/721d5228-5e5b-417d-a38f-d49fe10f3e8a)

**Reply by GerritRCE** [reply]
Hi jgromes,
According 
<img width="1023" alt="Screenshot 2023-09-20 at 19 12 10" src="https://github.com/jgromes/RadioLib/assets/34754703/b7d5d88a-1c0a-492b-af9b-4dd8c2911180">

**Reply by pe1evx** [reply]
Hi Jan. I have a module that has a dot at the 169Mhz mark. It's as I ordered.

**Reply by GerritRCE** [reply]
2.4.2. FrequencySynthesis
Table 52 Frequency Synthesizer Specification
in this table is specified that compleet range is available.
Gerrit

**Reply by pe1evx** [reply]
![IMG_4682](https://github.com/jgromes/RadioLib/assets/28441540/9f1537f7-7d56-4271-9d8f-06e7412a5c6d)

**Reply by jgromes** [reply]
@GerritRCE interesting, what is the source of the datasheet you showed? It looks quite differnet from one that I have found.

**Reply by GerritRCE** [reply]
dis is a link, I found more.
the chip used in these module are SX1276/77/78.
https://cdn.sparkfun.com/assets/learn_tutorials/8/0/4/RFM95_96_97_98W.pdf
Regards,
Gerrit

**Reply by GerritRCE** [reply]
This just frequency range chip, al filters used on the pcb are limited to a band example  169 / 433 / 868 / 915.
so the chip can handle all frequency, local regulation and pcb design filters limit the module to t smaller range.

**Comment by pe1evx** [comment]
Hi Jan,

I don't know if I made the changes on the best place (because of future upgrades) but I can receive now POCSAG at 172.450 Mhz with the RFM98.

Greetings Alex

**Reply by jgromes** [reply]
@pe1evx thanks for pointing this out, I changed the RadioLib RFM9x driver so that it will accept the whole frequency range. In addition, it lead to a simplification of the internals (because I can now consider RFM9x to be the exact same as SX127x), so that's an added bonus.

**Reply by pe1evx** [reply]
Thanks for the update
